docs/receiving.md

```diff
@@ -1,107 +1,117 @@
+## Payment Pointers
 
-The website specifies a [Payment Pointer](https://paymentpointers.org) from
-which the browser fetches connection details to
-[open a payment stream](#open-a-stream) with the receiver.
+A [payment pointer](https://paymentpointers.org) is a unique ID assigned to your
+payment account by your receiver.
 
-Since the Payment Pointer is a URL it is possible for the host of the Payment
-Pointer to redirect the browser to an alternate address using standard HTTP
-redirects.
+The expectation is that a receiver will issue one or more payment pointers to
+its customers and these customers will embed the pointer into the websites they
+want to monetize. The user's browser then fetches connection details from the
+pointer to open a payment stream with the receiver.
 
-The expectation is that a WM Receiver will issue one or more payment pointers to
-it's users and they will embed this into their websites that they wish to
-monetize.
+Payment pointers resolve to secure URLs, making it possible for receivers to
+redirect browsers to alternate addresses using standard HTTP redirects.
 
-One of the benefits of the Payment Pointer is it provides a level of indirection
-that allows the browser to protect the privacy of the user from the WM Sender.
+One benefit of payment pointers is that they provide a level of indirection that
+allows browsers to protect the privacy of users from their senders. Specifically,
+a browser uses a pointer in combination with a unique session ID to request
+a unique receiving address and shared secret from the website's receiver. Only
+the unique receiving address for the session is given  to the sender. This
+prevents the sender from correlating a payment pointer with a website.
 
-Specifically, the browser uses the Payment Pointer in combination with a unique
-session id to request a unique receiving address for payments for each session.
-Only the receiving address is given to the WM Sender.
+## Receiving Payments
 
-The WM Sender never sees the Payment Pointer (which is easy to correlate to the
-website), only the address which is unique for each session.
+Any entity that is able to accept payments on behalf of websites can be a
+receiver if it supports the necessary protocols. To accept a stream of payments
+on behalf of its users, a receiver **MUST** support the following protocols from
+the Interledger stack:
 
-## Receiving Payments
+* Interledger Protocol (ILP) - Establishes a micropayment stream and sends payments
+* Simple Payment Setup Protocol (SPSP) - Exchanges payment information between
+payee and payer to facilitate payment over ILP
+* STREAM Protocol - Establishes packet-switched connections between entities
+using the ILP
+
+### Interledger Protocol (ILP)
 
-Any entity that is able to accept payments on behalf of websites can be a Web
-Monetization receiver if it offers support for the necessary protocols.
+The ILP establishes a micropayments stream and sends payments. Interledger
```

Comment:
"The ILP routes individual micropayments" rather than "The ILP establishes a micropayments stream and sends payments." I think the current phrasing could be misleading because establishing the micropayments stream is a concern of STREAM